src/Client.php

```diff
@@ -43,7 +43,7 @@ public function getIndexer(): Indexer
         return $this->indexer;
     }
 
-    public function getIndex($name): Index
+    public function getIndex(string $name): Index
```

Comment:
We need PHP 7.4 here exactly @lyrixx.

The parent class type hint with `\Elastica\Index`, and we need `\JoliCode\Elastically\Index`. It's just one line but I can't get around it.